test/test_helper.rb

```diff
@@ -38,4 +40,40 @@ class ActionDispatch::IntegrationTest
     Capybara.reset_sessions!
     Capybara.use_default_driver
   end
-end
\ No newline at end of file
+end
+
+class JavascriptIntegrationTest < ActionDispatch::IntegrationTest
```

Comment:
I think it's worth following Rails 5 naming convention http://guides.rubyonrails.org/testing.html#system-testing, given we anyway migrate it.

So to name it `ApplicationSystemTestCase` instead, and place it into separate file under `test` dir